tests/materials/processing/strategies/test_admixture_strategy.py

```diff
@@ -1,10 +1,29 @@
+from werkzeug.datastructures import ImmutableMultiDict
 from slamd.materials.processing.models.admixture import Admixture
 from slamd.materials.processing.models.material import Costs
 from slamd.materials.processing.strategies.admixture_strategy import AdmixtureStrategy
 
```

Comment:
This comment is not connected to test_admixture_strategy but to test_base_materials_controller.py. As there are no changes in this file, I simply add a comment here. In the mentioned file we still use mocker to mock the service. However, as the methods are now all classmethods, we can also use monkeypatch there. Thus, for consistency reasons, we should replace mocker with monkeypatch in test_base_materials_controller.py